pytorch_toolkit/formula_recognition/tools/demo.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import yaml
+import os.path
+
+import cv2 as cv
+import torch
+import torch.optim as optim
+from im2latex.data.utils import create_list_of_transforms
+from im2latex.data.vocab import read_vocab
+from im2latex.models.im2latex_model import Im2latexModel
+
+
+class Im2latexDemo:
+    def __init__(self, config):
+        self.config = config
+        self.model_path = os.path.join(os.path.abspath("./"), config.get('model_path'))
+        self.vocab = read_vocab(os.path.join(os.path.abspath("./"), config.get('vocab_path')))
```

Comment:
In the config file I write relative paths to the vocab path & model path. In order to get the correct paths to these files, I have to resolve these paths relatively to parent directory of the demo script (because in the config file paths are written relatively to directory of the directory of the formula-recognition project).

Reply:
This is not the right way to get the path to parent directory of the demo script.

```python
import os.path as osp
root_dir = osp.dirname(osp.abspath(__file__))
self.model_path = osp.join(root_dir, config.get('model_path'))
```

Reply:
Thanks, done